src/rgw/rgw_lc.cc

```diff
@@ -29,10 +29,6 @@
 #include "rgw_sal.h"
 #include "rgw_sal_rados.h"
 
-// this seems safe to use, at least for now--arguably, we should
-// prefer header-only fmt, in general
-#undef FMT_HEADER_ONLY
-#define FMT_HEADER_ONLY 1
```

Comment:
this change does not belong to this commit or probably this PR.

Reply:
Nope, we had enough discussion about this one. ;-)
Looks like it is still in my patchset.